miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditServiceRecordController.java

```diff
@@ -0,0 +1,156 @@
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+package uk.ac.bbsrc.tgac.miso.webapp.controller;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.propertyeditors.CustomDateEditor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.SessionAttributes;
+import org.springframework.web.bind.support.SessionStatus;
+import org.springframework.web.servlet.ModelAndView;
+
+import uk.ac.bbsrc.tgac.miso.core.data.AbstractSequencerServiceRecord;
+import uk.ac.bbsrc.tgac.miso.core.data.SequencerReference;
+import uk.ac.bbsrc.tgac.miso.core.data.SequencerServiceRecord;
+import uk.ac.bbsrc.tgac.miso.core.factory.DataObjectFactory;
+import uk.ac.bbsrc.tgac.miso.core.manager.FilesManager;
+import uk.ac.bbsrc.tgac.miso.core.manager.RequestManager;
+import uk.ac.bbsrc.tgac.miso.sqlstore.util.DbUtils;
+
+@Controller
+@RequestMapping("/stats/sequencer/servicerecord")
+@SessionAttributes("serviceRecord")
+public class EditServiceRecordController {
+  
+  protected static final Logger log = LoggerFactory.getLogger(EditServiceRecordController.class);
+  
+  @Autowired
+  private RequestManager requestManager;
+
+  @Autowired
+  private FilesManager filesManager;
+
+  @Autowired
+  private DataObjectFactory dataObjectFactory;
+  
+  @Autowired
+  private JdbcTemplate jdbcTemplate;
+
+  public void setInterfaceTemplate(JdbcTemplate interfaceTemplate) {
+    this.jdbcTemplate = interfaceTemplate;
+  }
+
+  public void setRequestManager(uk.ac.bbsrc.tgac.miso.core.manager.RequestManager requestManager) {
```

Comment:
This may be an obvious answer, but why are you fully qualifying the RequestManager?